charts/seed-bootstrap/charts/vpa/templates/vpa-exporter.yaml

```diff
@@ -35,8 +35,8 @@ spec:
             cpu: 50m
             memory: 200Mi
           limits:
-            cpu: 200m
-            memory: 500Mi
+            cpu: 500m
```

Comment:
Can we remove the limits if vpa is enabled and keep them if it is disabled?

Reply:
Yes. I shall do that.

Reply:
Now the limits are removed entirely.
Like @wyb1 proposed, can you keep them if VPA is disabled?

Furthermore, what about the limits for `kube-state-metrics` @georgekuruvillak  and @wyb1?

Reply:
I propose simply putting an `if vpa enabled` around the limits. 
```yaml
{{ .Values.vpa.enabled }}
limits:
    cpu: 200m
    memory: 500Mi
{{ end }}
We should do this for the vpa-exporter and kube-state-metrics.

Reply:
makes sense. I shall do that.